Portfolio/index.css

```diff
@@ -1 +1,53 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+.main{
+    padding: 10px;
+    box-sizing: border-box;
+    background-image: radial-gradient( circle farthest-corner at 10% 20%,  rgba(255,94,247,1) 17.8%, rgba(2,245,255,1) 100.2% );
+}
+
+
+
+.asset{
+    height: 200px;
+    width: 200px;
+}
+
+.footer{
+    border-top: 2px solid white;
+    margin-top: 10px;
+}
+
+.form{
+    width: 50%;
+    place-items: center;
+    backdrop-filter: blur(4px);
+    border: 2px solid white;
+    border-radius: 10px;
+    padding: 10px;
+}
+
+
+input{
+    border: none;
+    margin-bottom: 5px;
+    border-radius: 2px;
+    width: 100%;
+    place-items: center;
+}
+
+#message{
+    height: 3vmax;
+}
+#formSubmit {
+    display: block;
+  margin: 0 auto;
+}
+
+.btn{
+    margin: 5px;
+    border: 2px solid white;
+    border-radius: 2px;
+    padding: 5px;
+    color: white;
+    background:transparent;

```

Comment:
Consider using a more specific class name instead of generic 'btn' to avoid potential conflicts with other styles.